homework1/src/app/game-state.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from "@angular/core";
+import { GameState } from "./game-state";
+
+@Injectable({
+  providedIn: "root",
+})
+export class GameStateService {
+  gameState: GameState = {
+    state: "start",
```

Comment:
you should create a separate enum with all the possible values for this state and use the values from it and not from hard-coded strings

Reply:
done